index.js

```diff
@@ -57,7 +57,7 @@ function addBookHandler(event) {
 
 // eslint-disable-next-line no-unused-vars
 function removeBookHandler(event) {
-  lib.remove(event.target.parentElement.id); location.reload();
+  lib.remove(event.target.parentElement.parentElement.id); location.reload();
```

Comment:
**[REQUIRED]** Make sure that the remove button is working fine.

Reply:
> **[REQUIRED]** Make sure that the remove button is working fine.

Hello @umararshadbut,

We had already fixed the problem with the remove button at the very beginning of this milestone. That was happening just because of the live link in the readme. Now the link has been updated.